packages/langium/src/references/scope.ts

```diff
@@ -18,7 +19,7 @@ export interface AstNodeDescription {
     node?: AstNode
     type: string // AstNodeType?
     name: string // QualifiedName?
-    documentUri: string // DocumentUri?
+    documentUri: URI // DocumentUri?
```

Comment:
The comment here was referring exactly to changing the type to something more concrete; it can be removed now.